### ISSUE TITLE
[v0.8.0] TypeError: this.getPane() is undefined

### ISSUE THREAD

**Issue by 1Luc1** [vue-leaflet/vue-leaflet#265]
After switching from v0.7.0 to [v0.8.0](https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.8.0) the following error occourred with the quickstart example:
leaflet version: 1.9.3

<details><summary>TypeError: this.getPane() is undefined</summary>

```
Uncaught (in promise) TypeError: this.getPane() is undefined
    _initContainer GridLayer.js:361
    onAdd GridLayer.js:157
    _layerAdd Layer.js:114
    whenReady Map.js:1477
    addLayer Layer.js:172
    addLayer LMap.vue:240
    wrapper utils.js:116
    setup LTileLayer.vue:32
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    flushJobs runtime-core.esm-bundler.js:397
    promise callback*queueFlush runtime-core.esm-bundler.js:282
    queueJob runtime-core.esm-bundler.js:276
    effect runtime-core.esm-bundler.js:5743
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:387
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    trigger reactivity.esm-bundler.js:355
    set2 reactivity.esm-bundler.js:526
    setup LMap.vue:362
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    render2 runtime-core.esm-bundler.js:6263
    mount runtime-core.esm-bundler.js:4456
    mount runtime-dom.esm-bundler.js:1607
    <anonymous> main.ts:28
[GridLayer.js:361:7](http://localhost:8080/node_modules/leaflet/src/layer/tile/GridLayer.js)
```

</details>

**Comment by tft-kaare** [comment]
Can confirm. I've tried vue-leaflet-0.8.0 on clean installs of both Vue3 and Vuetify3 using yarn or npm, though with Vite — not webpack — as packaging tool. In both cases I've followed the simplest quick-start recommendation for setup with vue-router. Created a simple component/view with the example template provided in this package. Same error in both cases (and other more complex projects):

<details>
<summary>In clean Vue3 install:</summary>

```
Uncaught (in promise) TypeError: this.getPane() is undefined
    _initContainer GridLayer.js:361
    onAdd GridLayer.js:157
    _layerAdd Layer.js:114
    whenReady Map.js:1477
    addLayer Layer.js:172
    addLayer LMap.vue:240
    wrapper utils.js:116
    setup LTileLayer.vue:32
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    flushJobs runtime-core.esm-bundler.js:397
    promise callback*queueFlush runtime-core.esm-bundler.js:282
    queueJob runtime-core.esm-bundler.js:276
    effect runtime-core.esm-bundler.js:5743
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:387
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    trigger reactivity.esm-bundler.js:355
    set2 reactivity.esm-bundler.js:526
    setup LMap.vue:362
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    flushJobs runtime-core.esm-bundler.js:397
    promise callback*queueFlush runtime-core.esm-bundler.js:282
    queueJob runtime-core.esm-bundler.js:276
    effect runtime-core.esm-bundler.js:5743
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:387
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    set value reactivity.esm-bundler.js:1048
    finalizeNavigation vue-router.mjs:3334
    pushWithRedirect vue-router.mjs:3207
    promise callback*pushWithRedirect vue-router.mjs:3174
    push vue-router.mjs:3099
    install vue-router.mjs:3530
    use runtime-core.esm-bundler.js:4381
    <anonymous> main.js:12
```

</details>

<details>
<summary>In clean Vuetify3 install:</summary>

```
Uncaught (in promise) TypeError: this.getPane() is undefined
    _initContainer GridLayer.js:361
    onAdd GridLayer.js:157
    _layerAdd Layer.js:114
    whenReady Map.js:1477
    addLayer Layer.js:172
    addLayer LMap.vue:240
    wrapper utils.js:116
    setup LTileLayer.vue:32
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    flushJobs runtime-core.esm-bundler.js:397
    promise callback*queueFlush runtime-core.esm-bundler.js:282
    queueJob runtime-core.esm-bundler.js:276
    effect runtime-core.esm-bundler.js:5743
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:387
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    trigger reactivity.esm-bundler.js:355
    set2 reactivity.esm-bundler.js:526
    setup LMap.vue:362
    createHook runtime-core.esm-bundler.js:2739
    callWithErrorHandling runtime-core.esm-bundler.js:157
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    __weh runtime-core.esm-bundler.js:2713
    flushPostFlushCbs runtime-core.esm-bundler.js:343
    flushJobs runtime-core.esm-bundler.js:397
    promise callback*queueFlush runtime-core.esm-bundler.js:282
    queuePostFlushCb runtime-core.esm-bundler.js:304
    queueEffectWithSuspense runtime-core.esm-bundler.js:1585
    scheduler runtime-core.esm-bundler.js:1820
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:387
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    effect reactivity.esm-bundler.js:1137
    triggerEffect reactivity.esm-bundler.js:397
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:1003
    set value reactivity.esm-bundler.js:1048
    finalizeNavigation vue-router.mjs:3334
    pushWithRedirect vue-router.mjs:3207
    promise callback*pushWithRedirect vue-router.mjs:3174
    push vue-router.mjs:3099
    install vue-router.mjs:3530
    use runtime-core.esm-bundler.js:4381
    registerPlugins index.js:17
    <anonymous> main.js:18
```

</details>

**Comment by Nukz-zkuN** [comment]
Same

**Comment by mikeu** [comment]
Thank you for reporting this issue! I have just released a fix in v0.8.1, which is no longer exhibiting this error. Please let me know if it does not resolve the problem for you.

**Comment by 1Luc1** [comment]
Thanks works!
